stl/inc/xutility

```diff
@@ -3117,97 +3117,97 @@ public:
 
     _CONSTEXPR17 explicit _Array_const_iterator(pointer _Parg, size_t _Off = 0) : _Ptr(_Parg + _Off) {}
 
-    _NODISCARD _CONSTEXPR17 reference operator*() const {
+    _NODISCARD _CONSTEXPR17 reference operator*() const noexcept /* strengthened */ {

```

Comment:
Given that none of these operations are explicitly specified in the standard - container iterators are implementation-defined - I don't think it makes sense to annotate these with `/* strengthened */` (applies to every occurrence of `/* strengthened */`).